### PR TITLE
Works and about sections

### PR DESCRIPTION
**Pull request by batoolfatima2135**
I have created two more sections
1. Work section: To display all of my previous work
2. About section: To tell about myself and my known languages, skills, and frameworks.
Sections are made using just HTML and CSS just similar to my selected FIGMA template fulfilling all the requirements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,6 +37,177 @@ <h1 class="intro">I’m a software developer</h1>
             </button>
         </section>
     </section>
+    <section class="works_section">
+        <h1>My Recent <br> Works</h1>
```
Comment (christianbiring1) [inline index.html]: - [ ] The page is supposed to only have one heading element `h1` for semantic reasons, Kindly you may consider using `h2` ...`hx`

File: index.html [hunk 2]
```diff
@@ -37,6 +37,177 @@ <h1 class="intro">I’m a software developer</h1>
...
+                        <button>See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+      
+    </section>
+    <section class="about_section">
+        <div class="about-details">
+            <h1>About me</h1>
```
Comment (christianbiring1) [inline index.html]: - [ ] The page is supposed to only have one heading element `h1` for semantic reasons, Kindly you may consider using `h2` ...`hx`

File: index.html [hunk 3]
```diff
@@ -37,6 +37,177 @@ <h1 class="intro">I’m a software developer</h1>
...
+            <button >Get my resume</button>
+        </div>
+        
+        <div class="about-tech">
+            <div class="languages">
+                <div class="icon-language">
+                    <img src="images/about/icon-languages.png" alt="icon-languages">
+                </div>
+                <h2>Languages</h2>
+                <div class="list">
```
Comment (christianbiring1) [inline index.html]: - [ ] For semantic reasons, Kindly make sure your technologies categories("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
```html
<ul>
<li>Languages</li>
...
<li>Framewors</li>
...
</ul>
```

File: style.css [hunk 4]
```diff
@@ -99,3 +102,216 @@ p {
   display: flex;
   justify-content: center;
 }
+
+.works_section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (christianbiring1) [inline style.css]: - [ ] The main container of the projects is supposed to have `display:grid`. Kindly fix that

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @batoolfatima2135 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :1st_place_medal: 
- [ ] Good looking UI :art: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ABDULALI3468 — APPROVED** [review]
Hi @batoolfatima2135 👍✅  

# APPROVED 💯 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
☑️Implemented changes asked by the previous code reviewer.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**